src/gsTensor/gsTensorBasis.hpp

```diff
@@ -313,9 +313,9 @@ gsMatrix<unsigned> gsTensorBasis<d,T>::allBoundary() const
        {
        bdofs= dofs / size(k);
        r-=  bdofs;
-       res->block(r,0, bdofs, 1) = *safe( this->slice(i, size(k) - 1) );
+       res->block(r,0, bdofs, 1) = ( this->slice(i, size(k) - 1) );
```

Comment:
same as above

Reply:
It's code that is for more then 3 years out-commented.

Reply:
It could be removed then.

Reply:
Ahh.. I see, it should be some type of example. Ignored yesterday every out-commended code and looked fast over it without any recognition.

Reply:
Just remove this commented part, it is too old..